app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="container flex food-placeholder">
+<div class="food flex">
+  <%= link_to new_recipe_path do %>
+    <button>Add Recipe</button>
+  <% end %>
+
+  <% @recipes.each do |recipe| %>
+  <div class='recipe flex'>
+    <div class="recipe-details flex">
+      <h2><%= link_to recipe.name, recipe_path(recipe) %></h2>
+      <%= button_to "Remove", recipe_path(recipe.id),data: { turbo_method: :delete, turbo_confirm: "Are you sure?"}, method: :delete, class: "delete" %>
+    </div>
+    <p><%= recipe.description.truncate_words(50, omission: '...') %></p>
+  </div>
+  <% end %>
```

Comment:
- [ ] Kindly inside this view only show the recipes of the logged user as per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/buisness_requirements.md#:~:text=Should%20display%20a%20list%20of%20recipes%20created%20by%20the%20logged%2Din%20user%20as%20in%20the%20wireframe.).